<launch>

    <arg name="strapdown_config_file" default="$(find etddf)/config/strapdown.yaml" />
    <arg name="starting_position_enu" default="[-3,8,0]" /> <!-- [x,y,z] -->
    <arg name="starting_yaw_enu" default="3.14" />
    <arg name="wait_on_startup" default="0" />

    <node name="strapdown" pkg="etddf" type="strapdown_ins.py" output="screen">
        <rosparam command="load" file="$(arg strapdown_config_file)" />
        <rosparam param="starting_position" subst_value="True">$(arg starting_position_enu)</rosparam> 
        <param name="starting_yaw" value="$(arg starting_yaw_enu)" />
        <param name="wait_on_startup" value="$(arg wait_on_startup)" />
    </node>

</launch>